cmd/soroban-cli/src/utils/contract-init-template/contracts/hello_world/Cargo.toml.removeextension

```diff
@@ -1,7 +1,7 @@
 [package]
-name = "soroban-hello-world-contract"
+name = "hello-world"
 version = "0.0.0"
-authors = ["Stellar Development Foundation <info@stellar.org>"]
+authors = ["John Doe <john@example.com>"]
```

Comment:
Suggest changing the package part of the file to only the following which is more consistent with what `cargo init` does:
```
[package]
name = "hello-world"
version = "0.0.0"
edition = "2021"
publish = false
```
This means dropping `authors`, `license`, `rust-version`.

Reply:
done!